src/scripts/main.js

```diff
@@ -1 +1,84 @@
-'use strict';
+const body = document.querySelector('body');
+
+function firstPromise() {
+  try {
+    let a = 0;
+    const timeOut = setTimeout(() => {
+      const errorMessage =
+        '<div data-qa="notification" class="error">' +
+        'First promise was rejected</div>';
+
+      body.insertAdjacentHTML('afterbegin', errorMessage);
+      a = 2;
+    }, 3000);
```

Comment:
The `setTimeout` here is intended to simulate a promise rejection, but it will execute regardless of any other conditions. Consider incorporating promise rejection into the logic explicitly.